.github/docker-images/base-images/debian-ubuntu/Dockerfile

```
@@ -12,9 +12,9 @@ RUN apt update && apt upgrade -y && \
 RUN mkdir /home/dependencies
 
 WORKDIR /home/dependencies
-RUN wget https://www.zlib.net/zlib-1.2.13.tar.gz -O /tmp/zlib-1.2.13.tar.gz && \
-	tar xzvf /tmp/zlib-1.2.13.tar.gz && \
-	cd zlib-1.2.13 && \
+RUN wget https://github.com/madler/zlib/releases/download/v1.3/zlib-1.3.tar.gz -O /tmp/zlib-1.3.tar.gz && \
+	tar xzvf /tmp/zlib-1.3.tar.gz && \
+	cd zlib-1.3 && \
```

You may be breaking compatibility by changing the lib version, what do you think about using the same version.

```suggestion
RUN wget https://github.com/madler/zlib/archive/v1.2.13.tar.gz -O /tmp/zlib-1.2.13.tar.gz && \
	tar xzvf /tmp/zlib-1.2.13.tar.gz && \
	cd zlib-1.2.13 && \
```

---

Ok, changed. Thanks